HW_1/hw_task_1.py

```diff
@@ -0,0 +1,23 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. Дано a, b, c —
+# стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой
+# двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника
+# с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним,
+# равнобедренным или равносторонним.
+
+def triangle(a, b, c):
+    if a < (b+c) and b < (a+c) and c < (a+b):
+        print("Треугольник с заданными сторонами существует")
+        if a == b == c:
+            print("Треугольник равносторонний")
+        elif a == b or a == c or b == c:
+            print("Треугольник равнобедренный")
+        else:
+            print("Треугольник разносторонний")
+    else:
+        print("Треугольник с заданными сторонами не существует")
```

Comment:
Согласно концепции повторного использования кода, если функция планируется вызываться всего 1 раз, то определять ее вовсе не нужно. Но раз уж вы используете функцию, то используйте ее по полной
```suggestion
    if a < (b+c) and b < (a+c) and c < (a+b):
        print("Треугольник с заданными сторонами существует")  # без этого print можно обойтись
        if a == b == c:
            return "Треугольник равносторонний"
        if a == b or a == c or b == c:
            return "Треугольник равнобедренный"

        return "Треугольник разносторонний"

    return "Треугольник с заданными сторонами не существует"
```